### PR TITLE
Implements `--title` option

### LINKED ISSUE #161: add option to display a title
It would be nice to be able to supply an additional option ,e.g. `--title=testset`, that would then be displayed as title of the figure.

### PR DESCRIPTION
**Pull request by abelsiqueira**
Closes #161.

### CODE REVIEW

**Comment by abelsiqueira** [comment]
@rgaiacs I rebased this PR to work with the translations.

File: perprof/main.py [hunk 1]
```diff
@@ -186,6 +187,8 @@ def set_arguments(args):
                     'is no primal feasibility to check.'))
     parser.add_argument('--infeasibility-tolerance', type=float, default=1e-4,
             help=_('Tolerance for the primal and dual infeasibilities'))
+    parser.add_argument('--title', type=str,
+            help=_('Sets the performance profile title'))
```
Comment (rgaiacs) [inline perprof/main.py]: I prefer something like "Set the title to be show on top of the performance profile".

File: perprof/main.py [hunk 2]
```diff
@@ -186,6 +187,8 @@ def set_arguments(args):
                     'is no primal feasibility to check.'))
     parser.add_argument('--infeasibility-tolerance', type=float, default=1e-4,
             help=_('Tolerance for the primal and dual infeasibilities'))
+    parser.add_argument('--title', type=str,
```
Comment (rgaiacs) [inline perprof/main.py]: You can add `default="Performance Profile"` and avoid all the `if`. :smile:

**Comment by rgaiacs** [comment]
@abelsiqueira Could you try use the `default`? After that I think this is ready to merge.

**Comment by abelsiqueira** [comment]
I tried setting `default=_('Performance Profile')` but the translation isn't working.

**Comment by rgaiacs** [comment]
@abelsiqueira Could you merge https://github.com/abelsiqueira/perprof-py/pull/2 ?

~~~
$ perprof --demo --tikz -f -o /tmp/p.pdf --lang pt_BR
$ perprof --demo --tikz -f -o /tmp/p.pdf
$ perprof --demo --tikz -f -o /tmp/p.pdf  --title foo
$ perprof --demo --tikz -f -o /tmp/p.pdf --lang pt_BR --title foo
~~~

is OK.

**Pull request merged**